ecommerce-ui/src/ShoppingCart.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import ShoppingItem from './ShoppingItem'
+import { Button, Card, Badge } from '@material-ui/core';
+import { lightGreen } from '@material-ui/core/colors';
+import  ShoppingCartIcon  from '@material-ui/icons/ShoppingCart';
+
+
+const styles = theme => ({
+    background: {
+        background: lightGreen[200]         
+    },
+    button: {
+        marginTop: 10,
+        marginBottom: 10,
+        float: "left",
+        position: "relative",
+        right:"-1%"
+      },
+
+    cartButton: {
+        marginTop: 10,
+        marginBottom: 10,
+        float: "right",
+        position: "relative",
+        left:"-5%"
+    },
+
+    cartBadge : {
+        marginTop: 10,
+        marginBottom: 10,
+        float: "right",
+        position: "relative",
+        left:"-18%"
+    }
+  });
+
+class ShoppingCart extends React.Component {
+
+    toggleList()
+    {
+       var listdivEl = document.getElementById("shoppingList");
```

Comment:
Alternatively, you can also use React refs to get a reference to an existing element. https://reactjs.org/docs/refs-and-the-dom.html

Also, this would be a good case to move the state of the list toggle to the parent component.